File: clients/linux/Mashr.MediaDeck.Linux/SecurityPolicy.cs
using System.Net;

namespace Mashr.MediaDeck.Linux;

public static class SecurityPolicy
{
    public const int CompanionPort = 43821;
    public const int DiscoveryPort = 43822;
    public const bool NetworkTransportEnabled = false;

    public static bool IsSafeScaffoldBind(IPAddress address) => IPAddress.IsLoopback(address);
}

File: clients/macos/Sources/MashrMediaDeckMac/SecurityPolicy.swift
import Foundation

enum SecurityPolicy {
    static let companionPort: UInt16 = 43821
    static let discoveryPort: UInt16 = 43822
    static let networkTransportEnabled = false

    static func isSafeScaffoldBind(_ host: String) -> Bool {
        host == "127.0.0.1" || host == "::1" || host.lowercased() == "localhost"
    }
}

